### ISSUE TITLE
attention mask issue

### ISSUE THREAD

**Issue by husai033** [aub-mind/arabert#51]
I am trying to run the code for  Text-classification/Sentiment Analysis with Hyper Parameter tuning available on 
https://colab.research.google.com/drive/1P9iQHtUH5KUbTVtp8B4-AopZzEEPE0lw?usp=sharing
without hyper parameter optimization using the regular training code, but I got the following error when I run trainer.train() 

<img width="775" alt="Screen Shot 2021-01-16 at 2 11 01 AM" src="https://user-images.githubusercontent.com/12014572/104805724-2f9af380-57a0-11eb-8386-1a3cc1748283.png">

**Comment by WissamAntoun** [comment]
I think the error is in the `self.label_map`, which dataset are you using?

**Comment by husai033** [comment]
the L-HSAB 
https://github.com/Hala-Mulki/L-HSAB-First-Arabic-Levantine-HateSpeech-Dataset
I used this label_map to convert the dataset to binary labels
label_map = {
    'abusive' : 1,
    'hate' : 1,
    'normal' : 0
}

**Comment by WissamAntoun** [comment]
then the issue might be from the `self.target` not being a list

**Comment by husai033** [comment]
Thank you so much, it works perfectly now. I just fixed the labels.
